tests/test_stack_report_helper.py

```diff
@@ -174,30 +182,66 @@ def test_populate_key_count_failure():
         assert e.value == 'TypeError("unhashable type: \'list\'",)'
 
 
-def test_S3helper():
+def test_s3helper():
```

Comment:
@miteshvp is this change required?

Reply:
Based on the pythonic way, definitions are lowercase